site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: tutorials
+menu: tutorials
+title: Introduction to Vega-Lite
+permalink: /tutorials/getting_started.html
+---
+
+In this tutorial, you will learn how to write a visualization specification in Vega-Lite. You will start with a simple visualization, which you will then extend the encoding and define additional properties. Lastly, you will learn how to embed the final visualization on a web page. There are similar tutorials for [D3](http://bost.ocks.org/mike/bar/) and [Vega](https://github.com/vega/vega/wiki/Tutorial).
```

Comment:
I would say this at the end of "spec" section.   Saying here is so distracting.  Do you want people to keep reading our stuff or jump to see those pages. 

Let's keep them reading our stuff first and once they see how simple our stuff is --  it's not too late to ask them to take a look at D3 / Vega (and see that we're more concise).